scripts/config.py

```diff
@@ -217,7 +217,8 @@ def realfull_adapter(_name, active, section):
     'MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND', # build dependency (valgrind headers)
     'MBEDTLS_X509_REMOVE_INFO', # removes a feature
     'MBEDTLS_SSL_RECORD_SIZE_LIMIT', # in development, currently breaks other tests
-    'MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED' # influences SECP256R1 KeyGen/ECDH/ECDSA
+    'MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED', # influences SECP256R1 KeyGen/ECDH/ECDSA
+    'MBEDTLS_ECP_WITH_MPI_UINT' # disables the default ECP and is experimental
```

Comment:
The list should be, and used to be, but no longer is, in alphabetical order, and with a comma on the last element. Since this is preexisting and we want to move quickly, not a blocker.